blueprints/ember-electron/index.js

```diff
@@ -24,96 +21,63 @@ class EmberElectronBlueprint extends Blueprint {
-        logger.message('https://github.com/felixrieseberg/ember-electron');
-      });
+    return this._installElectronTooling(logger)
+      .then(() => this._createResourcesDirectories(logger))
+      .then(() => this._ensurePackageJsonConfiguration(logger));
   }
 
-  _installDependencies(logger) {
-    // Since addPackagesToProject() doesn't give us fine-grained control over save vs. save-dev,
-    // or the exact flag, we use:
-    let task = this.taskFor('npm-install');
-
-    logger.startProgress('Installing electron tools (electron-forge + electron-protocol-serve)');
-
-    // There are two parts of electron-forge's import command that are relevant to us.
-    // One is installing dependencies, and the other is setting up the electron-forge configuration.
-    // Rather than embedding the forge configuration inline in package.json
-    // we want it in an external file, so we install the dependencies here ourselves
-    // (including our own dependencies), and add an entry to package.json
-    // pointing the forge config to the config file that is copied from the blueprint files.
-
-    const { devDeps, exactDevDeps, deps } = require('electron-forge/dist/init/init-npm');
-
-    return task.run({
-      'save-dev': true,
-      verbose: false,
-      packages: devDeps
-    }).then(() => task.run({
-      // Dependencies we need to keep locked at specific versions
-      'save-dev': true,
-      'save-exact': true,
-      verbose: false,
-      packages: exactDevDeps
-    })).then(() => task.run({
-      // Production dependencies
-      save: true,
-      verbose: true,
-      packages: ['electron-protocol-serve@1.1.0', ...deps]
-    }));
+  _installElectronTooling(logger) {
+    logger.startProgress('Installing electron build tools');
+
+    return efImport({
+      updateScripts: false,
+      outDir: 'electron-out',
+    })
+      .then(() => this.addPackageToProject('devtron', '^1.4.0'))
+      // n.b. addPackageToProject does not let us save prod deps, so we task
+      .then(() => this.taskFor('npm-install').run({
+        save: true,
+        verbose: false,
+        packages: ['electron-protocol-serve@1.1.0'],
+      }))
+      .then(() => logger.message('Installed electron build tools'));
   }
 
-  _createResourcesDirectories() {
-    const platforms = ['', 'darwin', 'linux', 'win32'];
-    let promises = platforms.map((platform) => {
-      let gitKeepPath = path.join(this.options.project.root, 'ember-electron', `resources${platform ? '-' : ''}${platform}`, '.gitkeep');
-      return ensureFile(gitKeepPath);
-    });
+  _createResourcesDirectories(logger) {
+    const ensureFile = denodeify(fs.ensureFile);
 
-    return all(promises);
+    logger.startProgress('Creating ember-electron resource dirs');
+
+    let rootDir = this.options.project.root;
+    let promises = [null, 'darwin', 'linux', 'win32']
+      .map((platform) => platform ? `resources-${platform}` : 'resources')
```

Comment:
To my eye, this step makes it harder to read because I have to think to figure out what directories we are creating. Why not just `['resources', 'resources-darwin', 'resources-linux', 'resources-win32'].map()`?